pai-management/bootstrap/frameworklauncher/frameworklauncher-configuration/frameworklauncher.yml

```diff
@@ -21,6 +21,8 @@
 zkConnectString: {ZOOKEEPER_ADDRESS}:2181
 zkRootDir: /Launcher
 hdfsRootDir: /Launcher
+rootAdminUsers: !!set
```

Comment:
@Gerhut if you want to enable ACL, please enable below in this file:
webServerAclEnable 
webServerAclIgnoreWithoutNamespace